osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -473,5 +506,35 @@ private void load()
                 Loading = true;
             }
         }
+
+        private class MutedNotification : SimpleNotification
+        {
+            public MutedNotification()
+            {
+                Text = "Your music volume is set to 0%! Click here to restore it.";
+            }
+
+            public override bool IsImportant => true;
+
+            public override bool RequestsFocus => true;
```

Comment:
What's this?

Reply:
Boolean overrides so that the notification is focused. The RequestsFocus can be removed as it's from Drawable. What's wrong with it?

Reply:
@Craftplacer Is there an issue with removing it? if no, then this is just redundant.